ipfs/using_ipfs_with_caver/src/main/java/Boilerplate.java

```diff
@@ -0,0 +1,130 @@
+    private static String senderPrivateKey = ""; // e.g. "0x42f6375b608c2572fadb2ed9fd78c5c456ca3aa860c43192ad910c3269727fc7"
+    private static String recipientAddress = ""; // e.g. "0xeb709d59954f4cdc6b6f3bfcd8d531887b7bd199"
+
+    public static void main(String[] args) {
+        try {
+            loadEnv();
+            run();
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static String objectToString(Object value) throws JsonProcessingException {
+        ObjectWriter ow = new ObjectMapper().writer().withDefaultPrettyPrinter();
+        return ow.writeValueAsString(value);
+    }
+
+    public static void loadEnv() {
+        Dotenv env;
+        String workingDirectory = System.getProperty("user.dir");
+        Path workingDirectoryPath = Paths.get(workingDirectory);
+        String projectRootDirectory = "caver-java-examples";
+        String currentDirectoryName = workingDirectoryPath.getName(workingDirectoryPath.getNameCount() - 1).toString();
+        String envDirectory = currentDirectoryName.equals(projectRootDirectory) ?
+                workingDirectoryPath.toString() :
+                workingDirectoryPath.getParent().getParent().toString();
+
+        // Read `/path/to/caver-java-examples/.env` file.
+        env = Dotenv.configure().directory(envDirectory).load();
+
+        nodeApiUrl = nodeApiUrl.equals("") ? env.get("NODE_API_URL") : nodeApiUrl;
+        accessKeyId = accessKeyId.equals("") ? env.get("ACCESS_KEY_ID") : accessKeyId;
+        secretAccessKey = secretAccessKey.equals("") ? env.get("SECRET_ACCESS_KEY") : secretAccessKey;
+        chainId = chainId.equals("") ? env.get("CHAIN_ID") : chainId;
+        senderAddress = senderAddress.equals("") ? env.get("SENDER_ADDRESS") : senderAddress;
+        senderPrivateKey = senderPrivateKey.equals("") ? env.get("SENDER_PRIVATE_KEY") : senderPrivateKey;
+        recipientAddress = recipientAddress.equals("") ? env.get("RECIPIENT_ADDRESS") : recipientAddress;
+    }
+
+    public static void run() throws Exception {
+        HttpService httpService = new HttpService(nodeApiUrl);
+        httpService.addHeader("Authorization", Credentials.basic(accessKeyId, secretAccessKey));
+        httpService.addHeader("x-chain-id", chainId);
+        Caver caver = new Caver(httpService);
+
+        // Set connection with IPFS Node
+        caver.ipfs.setIPFSNode("ipfs.infura.io", 5001, true);
+        // `ipfs.txt` is located at `caver-java-examples/ipfs/using_ipfs_with_caver/resources`.
+        File testFile = new File("resources/ipfs.txt");
+        if (testFile.exists() == false) {
+            // Handles when you run this Boilerplate as sub-module using IDE.
+            testFile = new File("ipfs/using_ipfs_with_caver/resources/ipfs.txt");
+            if (testFile.exists() == false) {
+                throw new Exception("Cannot find ipfs.txt testFile.");
```

Comment:
```suggestion
                throw new Exception("Cannot find ipfs.txt testFile.");
```
Please throw exception that fit the meaning. 
Exception class is the top-level class that handles exception.

Reply:
@sirano11 
Ok. I'll update this PR. 
[FileNotFoundException](https://docs.oracle.com/javase/7/docs/api/java/io/FileNotFoundException.html) would be suitable.

Reply:
@sirano11 
I updated it [here](https://github.com/klaytn/caver-java-examples/pull/5/commits/43702b476149868dc25d747a3793f98fe2594b1e) :)